frontend/src/pages/Projects/Project/CloseOut/index.tsx

```diff
@@ -0,0 +1,174 @@
+import { Button, Grid, LinearProgress, Typography } from "@mui/material";
+import { EditForm } from "components/EditForm";
+import { ReadForm } from "components/ReadForm";
+import { useFormSubmit } from "hooks/useFormSubmit";
+import React, { useState } from "react";
+import { useQuery } from "react-query";
+import { useParams } from "react-router-dom";
+import { IEditFields } from "types";
+import { apiAxios } from "utils";
+import { Notify } from "./Notify";
+
+export const CloseOut = () => {
+  const { projectId } = useParams();
+  const [editMode, setEditMode] = useState(false);
+  const { handleUpdate, Notification } = useFormSubmit();
+
+  const getProject = async () => {
+    const project = await apiAxios().get(`projects/${projectId}`);
+    return project.data.data;
+  };
+
+  // Queries
+  // todo: Define a good type. "Any" type temporarily permitted.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const projectQuery: any = useQuery(`project - ${projectId}`, getProject, {
+    refetchOnWindowFocus: false,
+    retryOnMount: false,
+    refetchOnReconnect: false,
+    retry: false,
+    staleTime: Infinity,
+  });
+
+  const readFields = [
```

Comment:
so we should maybe split these files out the readFields and editFields, there is two ways that this has been done one file for each read / edit or just one file for both fields.js (Admin/resources).  @mhaswell-bcgov @kshapka-bcgov @ASpiteri-BCGov we should decide on the best way.

Reply:
I'll split this into a `fields.js` file.